competition-simulator/programming/time.md

```diff
@@ -1,62 +1,23 @@
 ---
 layout: page
-title: Simulated Time
+title: Robot Time
 ---
 
-# Simulated Time
+# Important methods
```

Comment:
Is there a need to change these titles? The topic here hasn't really changed -- it is still the time within the simulator which we're talking about. The fact that the user accesses the time values from the `Robot` object is very much an implementation detail.

Reply:
b82bc54